docs/elasticsearch/tasksapis.md

```diff
@@ -0,0 +1,171 @@
+```
+
+By including a task ID, you can get information specific to a particular task. Note that a task's ID consists of a node's identifying string and the task's numerical ID. For example, if your node's identifying string is `nodestring` and the task's numerical ID is `1234`, then your task's ID is `nodestring:1234`. You can find this information by running the `_tasks` API.
+
+```
+GET _tasks/<task_id>
+```
+
+You can also use the following parameters with your query.
+
+Parameter | Type | Description |
+:--- | :--- | :---
+nodes | list | A comma-separated list of node IDs or names to limit the returned information; use `_local` to return information from the node you're connecting to, leave empty to get information from all nodes
+actions | list | A comma-separated list of actions that should be returned. Leave empty to return all.
+detailed | boolean | Return detailed task information (default: false)
+parent_task_id | string | Return tasks with specified parent task id (node_id:task_number). Leave empty or set to -1 to return all.
+wait_for_completion | boolean | Wait for the matching tasks to complete. (default: false)
+group_by | enum | Group tasks by parent/child relationships or nodes. (default: nodes)
+timeout | time | Explicit operation timeout. (default: 30 seconds)
+master_timeout | time | Time to wait for a connection to the master node. (default: 30 seconds)
+
+For example, this sample request returns tasks currently running on a node named `odfe-node1`.
+
+```json
+GET /_tasks?nodes=odfe-node1
+{
+  "nodes": {
+      "Mgqdm0r9SEGClWxp_RbnaQ": {
+          "name": "odfe-node1",
+          "transport_address": "sample_address",
+          "host": "sample_host",
+          "ip": "sample_ip",
+          "roles": [
+              "data",
+              "ingest",
+              "master",
+              "remote_cluster_client"
+          ],
+          "tasks": {
+              "Mgqdm0r9SEGClWxp_RbnaQ:24578": {
+                  "node": "Mgqdm0r9SEGClWxp_RbnaQ",
+                  "id": 24578,
+                  "type": "transport",
+                  "action": "cluster:monitor/tasks/lists",
+                  "start_time_in_millis": 1611612517044,
+                  "running_time_in_nanos": 638700,
+                  "cancellable": false,
+                  "headers": {}
+              },
+              "Mgqdm0r9SEGClWxp_RbnaQ:24579": {
+                  "node": "Mgqdm0r9SEGClWxp_RbnaQ",
+                  "id": 24579,
+                  "type": "direct",
+                  "action": "cluster:monitor/tasks/lists[n]",
+                  "start_time_in_millis": 1611612517044,
+                  "running_time_in_nanos": 222200,
+                  "cancellable": false,
+                  "parent_task_id": "Mgqdm0r9SEGClWxp_RbnaQ:24578",
+                  "headers": {}
+              }
+          }
+      }
+  }
+}
+```
+
+## Task Cancelling
+
+After getting a list of tasks, you can cancel all cancellable tasks with this request:
```

Comment:
It's not a big deal, but our editor tells me it's: cancel, canceled, canceling, and cancellation. Following this logic, I believe we want cancelable.

Reply:
Again not a big deal, some context for what makes a task 'cancelable' might be useful.